src/public/modules/core/services/gtag.client.service.js

```diff
@@ -1,10 +1,23 @@
+const get = require('lodash/get')
+
 angular.module('core').factory('GTag', ['$rootScope', '$window', GTag])
 
 function GTag($rootScope, $window) {
   // Google Analytics tracking ID provided on signup.
   const GATrackingID = $window.GATrackingID
   let gtagService = {}
 
+  const getUserEmail = () => {
+    let user = $window.localStorage.getItem('user')
+    try {
+      let userDetails = JSON.parse(user)
+      let userEmail = get(userDetails, 'email', null)
+      return userEmail
+    } catch (error) {
+      return null
+    }
```

Comment:
Instead of copying the code, why don't you reuse the function from `auth.client.service`?
```
const user = Auth.getUser()
return user && user.email
```